bin/configs/python-flask.yaml

```diff
@@ -1,4 +1,4 @@
 generatorName: python-flask
-outputDir: samples/server/petstore/python-flask
+outputDir: samples/server/petstore/2_0/python-flask
 inputSpec: modules/openapi-generator/src/test/resources/2_0/python-flask/petstore.yaml
 templateDir: modules/openapi-generator/src/main/resources/python-flask
```

Comment:
we don't test 2.0 spec so no need to keep this sample folder and yaml config file